### DISCUSSION TITLE
On the author template should we improve this part of the site

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#34] [Enhancement Requests]
These templates are not often worked on and could do with reviewing

- Tweets not loading
- Some posts from the author not showing 

https://github.com/amnestywebsite/amnesty-wp-theme/issues/2566

**Comment by AmnestyAM** [comment]
https://github.com/amnestywebsite/amnesty-wp-theme/discussions/2142

**Comment by AmnestyAM** [comment]
an old discussion about making author pages translatable https://github.com/amnestywebsite/amnesty-wp-theme/discussions/2413

**Comment by AmnestyAM** [comment]
I discussed the possibility of extending authors functionality so we could have two author profile per post, this would help the Crisis Team, I said I would raise this at the next quarterly roadmap review.

**Comment by carolinejcourtney** [comment]
There is a lot of talk about author pages in the SEO space at the moment. I think search engines are starting to prefer content that has a named person as an author, rather than just generic brand content. This authored content is even better if the byline links to an author page which lists out the authors credentials and other work. 

To be honest, I think the Tweet Feed function here is probably less relevant than it was when these templates were first designed.
